src/types.ts

```diff
@@ -504,6 +512,11 @@ export interface LiquidityRewards {
   estimatedRewards: string,
 }
 
+export interface StakedDYDXLiquidity {
```

Comment:
feel like `StakedDYDX` could extend this interface which will require some interface naming changes

Reply:
updated to `StakedDYDX` and `StakedDYDXIncludingFloor extends StakedDYDX` - don't think this is the greatest naming so if you have a suggestion i'll probably take it